install_config/aggregate_logging.adoc

```diff
@@ -842,10 +842,10 @@ openshift_logging_es_pvc_size=104802308Ki
 openshift_logging_es_pvc_prefix=es-logging
 ----
 
-If using xref:../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#install-config-persistent-storage-dynamically-provisioning-pvs[dynamically provisioned PVs], the {product-title} logging installer creates PVCs that use the default storage class or the PVC specified with the `openshift_logging_elasticsearch_pvc_storage_class_name` parameter.
+If you are using xref:../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#install-config-persistent-storage-dynamically-provisioning-pvs[dynamically provisioned PVs], the {product-title} logging installer creates PVCs that use the default storage class or the one specified with the `openshift_logging_elasticsearch_pvc_storage_class_name` parameter.
 
```

Comment:
NOTE: Due to a merge error, these paragraphs were duplicated. i removed the extraneous paragraphs.